lib/home/watch_live/watch_live.dart

```diff
@@ -54,7 +54,7 @@ class _LobbyScreenState extends State<LobbyScreen> {
     super.initState();
     Timer(
       const Duration(seconds: 5),
-      () => navigator.pushReplacement(const WatchingLivestream()),
+      () => mounted ? navigator.pushReplacement(const WatchingLivestream()) : null,
     );
```

Comment:
I accidentally had this set to automatically start the livestream even if you leave the lobby 😅